### PR TITLE
Add moderation status behind a feature flag

### PR DESCRIPTION
**Pull request by michalszorad**
https://emurgo.atlassian.net/browse/YOMO-485
https://emurgo.atlassian.net/browse/YOMO-484

### CODE REVIEW

File: src/NftDetails/NftDetails.tsx [hunk 1]
```diff
@@ -26,7 +27,7 @@ export const NftDetails = () => {
   return (
     <FadeIn style={styles.container}>
       <ScrollView contentContainerStyle={styles.contentContainer}>
-        <NftImage nft={nft} />
+        {MODERATING_NFTS_ENABLED ? <ModeratedNftImage nft={nft} /> : <UnModeratedNftImage nft={nft} />}
```
Comment (banklesss) [inline src/NftDetails/NftDetails.tsx:30]: I think eslint is going to complain if `MODERATING_NFTS_ENABLED === true` is not used

File: src/Nfts/ImageGallery/ImageGallery.tsx [hunk 2]
```diff
@@ -30,7 +31,13 @@ export const ImageGallery = ({nfts = [], onSelect, onRefresh, isRefreshing}: Pro
       data={nfts}
       onRefresh={onRefresh}
       refreshing={isRefreshing}
-      renderItem={(nft) => <ModeratedImage onPress={() => onSelect(nfts.indexOf(nft))} nft={nft} key={nft.id} />}
+      renderItem={(nft) =>
+        MODERATING_NFTS_ENABLED ? (
```
Comment (banklesss) [inline src/Nfts/ImageGallery/ImageGallery.tsx:35]: I think eslint is going to complain if `MODERATING_NFTS_ENABLED === true` is not used

File: src/NftDetails/NftDetails.tsx (hunk 1 above)
Comment (michalszorad) [reply src/NftDetails/NftDetails.tsx:30]: eslint is not complaining as `MODERATING_NFTS_ENABLED` is expected to be boolean

**Comment by michalszorad** [comment]
@stackchain is the SonarQube checking coverage on all components and if so, should we start adding unit tests for components?

File: src/Nfts/ImageGallery/ImageGallery.tsx (hunk 2 above)
Comment (michalszorad) [reply src/Nfts/ImageGallery/ImageGallery.tsx:35]: eslint is not complaining as `MODERATING_NFTS_ENABLED` is expected to be boolean

**Comment by stackchain** [comment]
> @stackchain is the SonarQube checking coverage on all components and if so, should we start adding unit tests for components?

we are still configuring it.

**Pull request merged**